### ISSUE TITLE
[BUG] 403 - Cannot access /dashboard/home

### ISSUE THREAD

**Issue by Daerendor** [Leantime/leantime#1813]
**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**
Cannot access /dashboard/home when connected using OIDC auth.
The UI is also broken.

![image](https://github.com/Leantime/leantime/assets/54585742/ee54de62-293f-488f-bd57-6315c5e5f857)

**To Reproduce**
1. Clone leantime-docker
2. Set the envs:
```env
LEAN_OIDC_ENABLE = true
LEAN_OIDC_CLIEND_ID = 'client_id'
LEAN_OIDC_CLIEND_SECRET = 'client_secret'
LEAN_OIDC_PROVIDER_URL = 'https://provider'
```
Any other LEAN_OIDC_ should remain as is.
3. Open Leantime's URL and complete /install setup.
4. Log out
5. Log in with OIDC
6. Navigate to /dashboard/home (if not already done)
7. See the 403 error.

**Expected behavior**
You should be able to view the dashboard.

**Leantime Version**
v2.3.27

**Server**
Docker behind Nginx

**PHP / MySQL Version**
- PHP 8.1.21
- MySQL 8.1.0

**Additional context**
Using Nginx fix described in [this comment](https://github.com/Leantime/leantime/issues/1792#issuecomment-1666938350) - See #1792

**Comment by marcelfolaron** [comment]
There seems to be an issue with the dashboard in particular. Do you have any logs in your logs/error.log file?

**Comment by marcelfolaron** [comment]
This should now be fixed in 2.4-beta

**Comment by MultiMote** [comment]
@marcelfolaron I got the same error in 3.14.12 (docker behind nginx).

```
LEAN_OIDC_ENABLE = true
LEAN_OIDC_CLIENT_ID = ...
LEAN_OIDC_CLIENT_SECRET = ...
LEAN_OIDC_CREATE_USER = true
LEAN_OIDC_DEFAULT_ROLE = 20
## OpenID Connect setting for GitHub
LEAN_OIDC_PROVIDER_URL = https://token.actions.githubusercontent.com/
LEAN_OIDC_AUTH_URL_OVERRIDE = https://github.com/login/oauth/authorize
LEAN_OIDC_TOKEN_URL_OVERRIDE = https://github.com/login/oauth/access_token
LEAN_OIDC_USERINFO_URL_OVERRIDE = https://api.github.com/user,https://api.github.com/user/emails
LEAN_OIDC_SCOPES = user:email,read:user
LEAN_OIDC_FIELD_EMAIL = 0.email
LEAN_OIDC_FIELD_FIRSTNAME = name
```

`/help/firstLogin` redirects to `/errors/error403` for new users

![Image](https://github.com/user-attachments/assets/8b628a33-ff1d-411b-8c5b-e6b226a7f559)

**Comment by marcelfolaron** [comment]
Hey, thanks for pointing that out. 
I have a fix staged for the firstlogin issue in 3.4.12 this will go out here in the next couple days.
